addons/web/static/src/js/views/basic/basic_view.js

```diff
@@ -78,16 +79,19 @@ var BasicView = AbstractView.extend({
     _getFieldWidgetClass: function (viewType, field, attrs) {
         var FieldWidget;
         if (attrs.widget) {
-            FieldWidget = fieldRegistry.getAny([viewType + "." + attrs.widget, attrs.widget]);
+            const keys = [viewType + "." + attrs.widget, attrs.widget];
+            FieldWidget = owlFieldRegistry.getAny(keys) || fieldRegistry.getAny(keys);
             if (!FieldWidget) {
                 console.warn("Missing widget: ", attrs.widget, " for field", attrs.name, "of type", field.type);
             }
         } else if (viewType === 'kanban' && field.type === 'many2many') {
             // we want to display the widget many2manytags in kanban even if it
             // is not specified in the view
-            FieldWidget = fieldRegistry.get('kanban.many2many_tags');
+            FieldWidget = owlFieldRegistry.get('kanban.many2many_tags') ||
```

Comment:
Is this for future-proofing ?
Or is m2m tags converted ?

Reply:
future proof.
There are only 3 fieldRegistry uses in this file and they are all in this function.